65d6f17b0b6651007ded36dff9ba206fac8d7caa ensure grub-mkconfig ignores *.apk-new files
 util/grub-mkconfig.in | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/util/grub-mkconfig.in b/util/grub-mkconfig.in
index f8cbb8d7a..8b6501209 100644
--- a/util/grub-mkconfig.in
+++ b/util/grub-mkconfig.in
@@ -278,6 +278,10 @@ for i in "${grub_mkconfig_dir}"/* ; do
     *~) ;;
     # emacsen autosave files. FIXME: support other editors
     */\#*\#) ;;
+    # Copy of a template file from Alpine Grub package where the stock file
+    # was locally modified. Any such *.apk-new files should be ignored by
+    # grub-mkconfig.
+    *.apk-new) ;;
     *)
       if grub_file_is_not_garbage "$i" && test -x "$i" ; then
         echo
-- 
2.39.2

